_content/setup-wallets.md

```diff
@@ -0,0 +1,79 @@
+## Set Up Interledger accounts to send and receive test XRP on Testnet
```

Comment:
I'm not sure how I feel about capitalizing "testnet," because it makes it feel as though there's only a single/canonical testnet, which isn't necessarily true

Reply:
I have lowercased it now. Maybe there's an official name we can use for this particular implementation of a testnet. (E.g., something like the "Xpring-enabled Interledger testnet at wallet.ilpv4.dev ")